iothub_client/samples/iothub_client_device_twin_and_methods_sample/iothub_client_device_twin_and_methods_sample.c

```diff
@@ -345,10 +373,17 @@ static void iothub_client_device_twin_and_methods_sample_run(void)
 
             char* reportedProperties = serializeToJson(&car);
 
-            (void)IoTHubDeviceClient_GetTwinAsync(iotHubClientHandle, getCompleteDeviceTwinOnDemandCallback, NULL);
-            (void)IoTHubDeviceClient_SendReportedState(iotHubClientHandle, (const unsigned char*)reportedProperties, strlen(reportedProperties), reportedStateCallback, NULL);
             (void)IoTHubDeviceClient_SetDeviceMethodCallback(iotHubClientHandle, deviceMethodCallback, NULL);
             (void)IoTHubDeviceClient_SetDeviceTwinCallback(iotHubClientHandle, deviceTwinCallback, &car);
+            (void)IoTHubDeviceClient_GetTwinAsync(iotHubClientHandle, getCompleteDeviceTwinOnDemandCallback, NULL);
+
+            ThreadAPI_Sleep(3000);
```

Comment:
>ThreadAPI_Sleep [](start = 12, length = 15)

Why have we added the sleeps? They shouldn't be needed so i'd want to understand why they're in.

Reply:
It was part of my test to make sure that the order between twin commands was preserved. We can remove it.

---
In reply to: [328723736](https://github.com/Azure/azure-iot-sdk-c/pull/1211#discussion_r328723736) [](ancestors = 328723736)

Reply:
Removed

---
In reply to: [328730411](https://github.com/Azure/azure-iot-sdk-c/pull/1211#discussion_r328730411) [](ancestors = 328730411,328723736)